pkg/disttask/importinto/planner.go

```diff
@@ -465,6 +517,11 @@ func getSortedKVMetasOfEncodeStep(subTaskMetas [][]byte) (map[string]*external.S
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		if stepMeta.ExternalPath != "" && store != nil {
```

Comment:
```suggestion
		if stepMeta.ExternalPath != "" {
```
seems no need to check it, `InitDataStore` will fail in that case